test/journeys/specs/tap/widget-recents/index.js

```diff
@@ -33,7 +34,7 @@ describe(`Widget Recents`, () => {
 
   before(`load browser`, () => {
     browserLocal
-      .url(`/production/recents.html`)
+      .url(`https://code.s4d.io/widget-recents/latest/demo/index.html`)
```

Comment:
Is there a reason we aren't setting baseUrl to https://code.s4d.io/ if `process.env.TAP` is true?

Reply:
No. There's not a good reason. Good call.